packages/desktop/views/dashboard/settings/views/security/ChangePassword.svelte

```diff
@@ -159,13 +159,8 @@
         disabled={busy}
         submitHandler={isPasswordValid}
     />
-    <Checkbox
-        classes="mb-5"
-        label={localize('actions.exportNewStronghold')}
-        bind:checked={exportStrongholdChecked}
-        disabled={busy}
-    />
-    <div class="flex flex-row items-center">
+    <Checkbox label={localize('actions.exportNewStronghold')} bind:checked={exportStrongholdChecked} disabled={busy} />
```

Comment:
What do you think about improving the file in general and making it use space-y?

```html
<form id="form-change-password" on:submit|preventDefault={changePassword}>
    <Text type={TextType.h4} classes="mb-3">{localize('views.settings.changePassword.title')}</Text>
    <div class="flex flex-col w-full space-y-5">
        <Text type={TextType.p} secondary>{localize('views.settings.changePassword.description')}</Text>
        <PasswordInput
            error={localize(currentPasswordError)}
            bind:value={currentPassword}
            showRevealToggle
            placeholder={localize('general.currentPassword')}
            disabled={busy}
            submitHandler={isPasswordValid}
        />
        <PasswordInput
            error={newPasswordError}
            bind:value={newPassword}
            showRevealToggle
            strengthLevels={4}
            showStrengthLevel
            strength={passwordStrength.score}
            placeholder={localize('general.newPassword')}
            disabled={busy}
            submitHandler={isPasswordValid}
        />
        <PasswordInput
            bind:value={confirmedPassword}
            showRevealToggle
            placeholder={localize('general.confirmNewPassword')}
            disabled={busy}
            submitHandler={isPasswordValid}
        />
        <Checkbox
            label={localize('actions.exportNewStronghold')}
            bind:checked={exportStrongholdChecked}
            disabled={busy}
        />
        <div class="flex flex-row items-center">
            <Button
                size={ButtonSize.Medium}
                disabled={!currentPassword || !newPassword || !confirmedPassword || busy}
                type={HTMLButtonType.Submit}
            >
                {localize('views.settings.changePassword.title')}
            </Button>
            <Spinner {busy} message={localize(changeMessageLocale)} classes="ml-2" />
        </div>
    </div>
</form>
```

not super relevant, just a comment